schema/deploy/computed_columns/application_operator.sql

```diff
@@ -0,0 +1,22 @@
+-- Deploy ggircs-portal:computed_columns/application_operator to pg
+-- requires: tables/application
+
+begin;
+
+create or replace function ggircs_portal.application_operator(app ggircs_portal.application)
+returns setof ggircs_portal.organisation
```

Comment:
Shouldn't this return a single organisation?

Reply:
hm, I may not need application_operator or application_facility at all now that I think about it, maybe just the scalar-returning function is enough

Reply:
Indeed

Reply:
Yeah, they're unnecessary, the logic can go directly in the scalar-returning function. I'll remove them.